.env.example

```diff
@@ -1,4 +1,4 @@
-ENDPOINT_URL=http://localhost:4566
+SQS_ENDPOINT=http://localhost:4566
```

Comment:
Why are we replacing ENDPOINT_URL with SQS_ENDPOINT but not retaining ENDPOINT_URL and SQS_ENDPOINT as a new entry?